src/main/java/org/openrewrite/staticanalysis/RemoveExtraSemicolons.java

```diff
@@ -52,20 +59,33 @@ public Duration getEstimatedEffortPerOccurrence() {
     @Override
     public TreeVisitor<?, ExecutionContext> getVisitor() {
         return new JavaIsoVisitor<ExecutionContext>() {
+
             @Override
-            public J.Empty visitEmpty(J.Empty empty, ExecutionContext ctx) {
-                if (getCursor().getParentTreeCursor().getValue() instanceof J.Block) {
-                    return null;
+            public J.Block visitBlock(final J.Block block, final ExecutionContext executionContext) {
+                final Iterator<Statement> iterator = block.getStatements().iterator();
+                final List<Statement> result = new ArrayList<>(); //todo should this be arraylist?
```

Comment:
Is there anything left to do here?

Reply:
I was not sure if maybe in this tree structure you might prefer a different type of list implementation. Hence the comment. But from what I've seen by now I think this is fine